### ISSUE TITLE
The format of .xcodeproj file is modified after running `pod install` via CocoaPods 0.39

### ISSUE THREAD

**Issue by agger0207** [CocoaPods/CocoaPods#5416]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

1. Create a new project with Xcode and add a podfile 
2. Run `pod install` via CocoaPods 0.39

## What did you expected to happen?

Xcode Project File Format doesn't change after running `pod install`.

## What happened instead?

All pod dependencies are installed correctly.  And the project can be compiled successfully. But File format of project file is modified.

Default file format is as following:

```
// !$*UTF8*$!
{
	archiveVersion = 1;
	classes = {
	};
	objectVersion = 46;
	objects = {

/* Begin PBXBuildFile section */
		E6CCDDDC1CF7E1D000C16F93 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDDDB1CF7E1D000C16F93 /* main.m */; };
		E6CCDDDF1CF7E1D000C16F93 /* AppDelegate.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDDDE1CF7E1D000C16F93 /* AppDelegate.m */; };
		E6CCDDE21CF7E1D000C16F93 /* FirstViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDDE11CF7E1D000C16F93 /* FirstViewController.m */; };
		E6CCDDE51CF7E1D000C16F93 /* SecondViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDDE41CF7E1D000C16F93 /* SecondViewController.m */; };
		E6CCDDE81CF7E1D000C16F93 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = E6CCDDE61CF7E1D000C16F93 /* Main.storyboard */; };
		E6CCDDEA1CF7E1D000C16F93 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = E6CCDDE91CF7E1D000C16F93 /* Assets.xcassets */; };
		E6CCDDED1CF7E1D000C16F93 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = E6CCDDEB1CF7E1D000C16F93 /* LaunchScreen.storyboard */; };
		E6CCDDF81CF7E1D100C16F93 /* TestTests.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDDF71CF7E1D100C16F93 /* TestTests.m */; };
		E6CCDE031CF7E1D100C16F93 /* TestUITests.m in Sources */ = {isa = PBXBuildFile; fileRef = E6CCDE021CF7E1D100C16F93 /* TestUITests.m */; };
/* End PBXBuildFile section */
```

After `pod install`, the file format is as follwing:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>archiveVersion</key>
	<string>1</string>
	<key>classes</key>
	<dict/>
	<key>objectVersion</key>
	<string>46</string>
	<key>objects</key>
	<dict>
		<key>278F356B88C5209B45DE3B49</key>
		<dict>
			<key>explicitFileType</key>
			<string>archive.ar</string>
			<key>includeInIndex</key>
			<string>0</string>
			<key>isa</key>
			<string>PBXFileReference</string>
			<key>path</key>
			<string>libPods.a</string>
			<key>sourceTree</key>
			<string>BUILT_PRODUCTS_DIR</string>
		</dict>
		<key>2C7D879B8B5D3C7544E7D865</key>
```

The comparing result is attached.
![projectfileformat](https://cloud.githubusercontent.com/assets/5696474/15597107/84fac028-23ff-11e6-819f-c671ba832bd9.jpg)

BTW, this issue doesn't  happen with Cocoapods 0.38.2 and it doesn't happen with some old projects. But it seems that it is very easy to reproduce it with a new project created with Xcode 7.3.

## CocoaPods Environment

ℹ Please replace this with the output of `pod env`.
e.g. via `pod env | pbcopy`

## Project that demonstrates the issue

ℹ Please link to a project we can download that reproduces the issue.
You can delete this section if your issue is unrelated to build problems,
i.e. it's only an issue with CocoaPods the tool.

**Comment by nicolasbraun** [comment]
Already had this, simply move a file une your XCode browser structure and save the xcodeproj

**Comment by jcampbell05** [comment]
This is because CocoaPods favours using the XML format of the xcodeproj when it can as it is more easily mergable in git.

HoweverIf you are using a version of CocoaPods built for an older Xcode. If you use a newer Xcode it will default to the Plist file format.

If the CocoaPods version you are using is built for the version of Xcode you are using then it will generate the project in XML. CocoaPods 1.0 was updated to work with Xcode 7.3.1 hence why this happened, where as CocoaPods 0.39 only worked with older xcode versions so it fell back to the Plist format.

**Comment by manuyavuz** [comment]
@agger0207 This should be fixed in CocoaPods 1.0 version, see discussion in CocoaPods/Xcodeproj#356.

Can you try after updating to 1.0 version, and give feedback?

**Comment by agger0207** [comment]
@manuyavuz  It works correctly after updating to 1.0 version. Thanks very much !  

Would you merge the fix to CocoaPods 0.39 version in future ? Or should I always use the latest version instead ?

**Comment by jcampbell05** [comment]
@agger0207  You should always try to use the latest CocoaPods version with the latest Xcode. You will get this issue whenever they are out of step. I think there is work being done to remove this dependency but not sure how far they got.

cc/ @orta

**Comment by orta** [comment]
Aye, you should always aim to use the latest stable version, though, obviously this isn't a critical fix - so it shouldn't be forcing you to 1.0 ;).

**Comment by agger0207** [comment]
@jcampbell05 @orta  Thanks very much! I'll update my private libraries to support  CocoaPods 1.0 then I can always use the laster stable version :)
